Web/Views/Account/Accounts.cshtml

```diff
@@ -273,6 +283,8 @@
             url = addQueryParamToUrl(url, 'culture', $("#custom-locale").val());
             url = addQueryParamToUrl(url, 'providerId', $('#bank-provider').val());
             url = addQueryParamToUrl(url, 'bank_account_numbers', $('#bank-account-ids').val());
+            url = addQueryParamToUrl(url, 'accountsGroupId', $('#accounts-group-id').val());
+            url = addQueryParamToUrl(url, 'accountsGroupName', $('#accounts-group-name').val());
```

Comment:
What if only account group name or account group id is defined? Is it ok?

Reply:
If we only define groupName - but not groupID it will throw an error on the Aiia API. 
It is allowed though to define ID without name (it will be then reused)